### ISSUE TITLE
[UPDATED] stormpath.spring.security.enabled = false does not work

### ISSUE THREAD

**Issue by mraible** [stormpath/stormpath-sdk-java#1047]
Our Spring security integration should be disabled when `stormpath.spring.security.enabled = false`. Currently, the only way to disable it is by using a module that does not have it.

We want a project using our Stormpath Default library (`stormpath-default-spring-boot-starter`) to be able to disable our Spring Security integration via a property. That would be similar (or identical) to use one of our modules that does not have Spring Security at all.

**Comment by mraible** [comment]
Not a bug after talking with @lhazlewood. This property has to be added **and** the `stormpath()` hook has to be removed from your `SecurityConfiguration.java`. For example:

```java
@Configuration
public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
    @Override
    protected void configure(HttpSecurity http) throws Exception {
        /*http.apply(stormpath()).and()
            .authorizeRequests()
            .antMatchers("/api*//**").fullyAuthenticated()
            .antMatchers("*//**").permitAll();*/
        http.authorizeRequests().antMatchers("/**").permitAll();
    }
}
```

**Comment by mraible** [comment]
Re-opening as an enhancement. It'd be nice for developers if they didn't have to remove the `stormpath()` hook and they could just put `stormpath.disabled=true` in their `application.properties`.

**Comment by mrioan** [comment]
This issue is currently being implemented in the same branch where #1123 is being implemented
